src/MusicStore/StartupNtlmAuthentication.cs

```diff
@@ -1,149 +0,0 @@
-using System;
-using System.Security.Claims;
-using Microsoft.AspNetCore.Builder;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
-using Microsoft.EntityFrameworkCore;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Logging;
-using MusicStore.Components;
-using MusicStore.Models;
-
-namespace MusicStore
```

Comment:
This was always unused in this repo. We use this in the mainline aspnet/MusicStore to integration test auth